app/session.go

```diff
@@ -35,6 +36,19 @@ func (a *App) CreateSession(session *model.Session) (*model.Session, *model.AppE
 	return session, nil
 }
 
+func AddSessionToPool(session *model.Session) {
```

Comment:
I feel this naming doesn't really explain what's going on. We are essentially giving back the object to the pool. Would calling this `ReturnSessionToPool` be clearer?

Reply:
thanks, you are right I think `ReturnSessionToPool` makes more sense, changed as suggested.